Transfer/Tests/CreateTransferPresenterTests.swift

```diff
@@ -207,13 +211,31 @@ class CreateTransferTests: XCTestCase {
         Hyperwallet.clearInstance()
         HyperwalletTestHelper.programModel = .walletModel
         Hyperwallet.setup(HyperwalletTestHelper.authenticationProvider)
-        mockView.stopOnError = true
-        initializePresenter(createTransferResult: .failure, showAllAvailableSources: true)
-        assertResponse(isShowErrorPerformed: true,
-                       transferSourceCellConfigurationsCount: 3,
-                       transferSourceType: .user,
-                       selectedTransferDestination: true,
-                       isAvailableBalancePresent: false)
+        if error?.group == .business {
```

Comment:
no need of checking error.group, code takes care of tat. Unit test should not involve any business logic

Reply:
ok